baseplate/lib/live_data/zookeeper.py

```diff
@@ -2,12 +2,47 @@
 from typing import Optional
 
 from kazoo.client import KazooClient
+from kazoo.client import KazooState
 from kazoo.handlers.gevent import SequentialGeventHandler
+from prometheus_client import Counter
 
 from baseplate.lib import config
 from baseplate.lib.secrets import SecretsStore
 from baseplate.server.monkey import gevent_is_patched
 
+SESSION_LOST_TOTAL = Counter(
+    "zookeeper_client_session_lost",
+    "The number of times a Zookeeper session as been lost for a client.",
+)
+
+SESSION_SUSPENDED_TOTAL = Counter(
+    "zookeeper_client_session_suspended",
```

Comment:
Generally we try to have the last word be a unit that helps to imply the type of metric it is. For counters, typically that's `_total` so I'd suggest something like `zookeeper_client_session_suspended_total`. Without it, I had assumed that the next metric should have been a gauge without reading further.